conjur/init/init_controller.py

```diff
@@ -132,6 +164,6 @@ def write_conjurrc(self):
     @staticmethod
     def __ensure_overwrite_file(config_file):
         force_overwrite = input(f"File {config_file} exists. " \
-                                f"Overwrite [yes]: ")
+                                f"Overwrite? [yes]: ").strip()
         if force_overwrite != '' and force_overwrite.lower() != 'yes':
-            raise Exception(f"Not overwriting {config_file}")
+            raise Exception(f"Not overwriting {config_file}\n")
```

Comment:
error messages should never contain newlines.
If you need newlines in the output, add them where you write the output.